platform/android/java/lib/src/org/godotengine/godot/Godot.java

```diff
@@ -317,6 +322,17 @@ private boolean onVideoInit() {
 		return true;
 	}
 
+	/**
+	 * Returns true if the device meets the base requirements for Vulkan support, false otherwise.
+	 */
+	private boolean meetsVulkanRequirements(@Nullable PackageManager packageManager) {
+		if (packageManager == null) {
+			return false;
+		}
+
+		return Build.VERSION.SDK_INT >= Build.VERSION_CODES.N && packageManager.hasSystemFeature(PackageManager.FEATURE_VULKAN_HARDWARE_LEVEL, 1);
```

Comment:
What version are we comparing here?

Reply:
We're comparing the device SDK version, and this follows a similar logic as with the export logic:
If the device SDK version < Android N (which corresponds to API 24), we consider the device as not meeting the vulkan requirements.